editing/fx/passes/pact/approximate.py

```diff
@@ -74,10 +74,15 @@ def __init__(self, **kwargs):
         passes.append(ReplaceSequentialPatternPass(pattern, PACT_symbolic_trace, lambda x,y: PACTGELU(), f'_APPROXIMATE_GELU_PASS'))
         super().__init__(*passes, name_prefix='_APPROXIMATE_GELU_PASS')
 
+class ApproximateSiLUPass(SequentialPass):
```

Comment:
please rename to SiLU2GELUPass or whatever

Reply:
Done